extensions/adobe/experience/adcloud/syncedremarketingaudiences.example.1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "xdm:segmentId": "https://data.adobe.io/entities/aam-segment/4238827",
+  "xdm:segmentType": 3,
+  "xdm:timestamp": "2018-08-07T08:06:34.672Z",
+  "xdm:partnerDetails": [
+    {
+      "xdm:id": "2",
+      "xdm:data": "sample-conversion-id-123"
```

Comment:
the xdm:data seems like it should live elsewhere since this mapping between segment in AAM to external system like google for facebook is always the same from one AAM segment to one FB or Google Segment and divorced from the concept of when semgent was synced for an individual user

Reply:
You could structure this with partner details using segmemnt membership to store synced timestamp in the payload, but would need to repeat for each partner